cypress/e2e/models/administration/assessment_questionnaire/assessment_questionnaire.ts

```diff
@@ -2,10 +2,21 @@ import { click, clickByText, selectUserPerspective } from "../../../../utils/uti
 import { SEC, assessmentQuestionnaires } from "../../../types/constants";
 import { legacyPathfinderToggle } from "../../../views/assessmentquestionnaire.view";
 import { navMenu } from "../../../views/menu.view";
+import { button } from "../../../../e2e/types/constants";
+
+export interface AssessmentQuestionnaire {
+    fileName: string;
+}
 
 export class AssessmentQuestionnaire {
     public static fullUrl = Cypress.env("tackleUrl") + "/assessment";
 
+    constructor(
```

Comment:
@sshveta , thanks for the question. 
My plan is to have the following methods for all questionnaires including the Legacy Pathfinder questionnaire. I know the import method is not relevant to the Legacy Pathfinder questionnaire since it's already available , but the other methods are .
1) Import
2) Export 
3) View
4) Delete 
5)Enable
6)Disable